lib/routes-priv.js

```diff
@@ -60,6 +67,105 @@ class RoutesPriv {
     this.websocketOpts = { requestTimeout: requestTimeout * 1000 }
   }
 
+  onPrivateTosSetHttpRequest (req, res) {
+    this.validateUser(req, res, (err, data) => {
+      if (err) {
+        return this.handleAuthError(err, res)
+      }
+
+      const { actor } = data.parseReq
+
+      const meta = data.verify.meta
+      const user = meta.actions[0].authorization[0].actor
+      if (actor !== user) {
+        log('ERR_ACTOR_USER', actor, user)
+        return res.status(403).json({ error: 'ERR_INVALID_PAYLOAD' })
+      }
+
+      const ts = Date.now()
+      this.tosCollection.insertOne({
+        _id: TOS + '_' + user,
```

Comment:
Why is overwriting the mongo _id, i dont like this idea. 
Can create if not a new id field with a unique key for it.
Mongo _id has its own properties and methods https://mongodb.github.io/node-mongodb-native/2.0/tutorials/objectid/

Reply:
we just need a unique id. it will always stay unique

this way we can save some space in the db and it is indexed automatically